closed/autoconf/custom-hook.m4

```diff
@@ -441,36 +441,33 @@ AC_DEFUN([OPENJ9_CHECK_NASM_VERSION],
   OPENJ9_PLATFORM_EXTRACT_VARS_FROM_CPU($host_cpu)
 
   if test "x$OPENJ9_CPU" = xx86-64 ; then
-    AC_PATH_PROG([NASM],[nasm])
-    if test "x$NASM" != x ; then
-      AC_MSG_CHECKING([whether nasm version requirement is met])
+    UTIL_REQUIRE_PROGS([NASM] , [nasm])
```

Comment:
@dnakamura You should at least _try_ your changes before sharing here, otherwise you're wasting your time and mine. Spacing is significant in `m4` - the spaces on both sides of the comma must be removed so we have:
```
  UTIL_REQUIRE_PROGS([NASM],[nasm])
```
otherwise it fails like this:
```
Generating runnable configure script at /home/keithc/space/openj9/jdk15/build/.configure-support/generated-configure.sh
Using autoconf at /usr/bin/autoconf [autoconf (GNU Autoconf) 2.69]
stdin:98: error: AC_SUBST: `NASM ' is not a valid shell variable name
```